Makefile

```diff
@@ -4,7 +4,7 @@ DOCKER_REGISTRY_BASE 	?= influxdb
 IMAGE_TAG				?= $(shell git describe --tags --abbrev=0 --match '[0-9].*[0-9].*[0-9]' 2>/dev/null)
 IMAGE_NAME 				?= $(SERVICE_NAME)
 BUILD_IMAGE				?= ghcr.io/konpyutaika/docker-images/nifikop-build
-GOLANG_VERSION          ?= 1.18
+GOLANG_VERSION          ?= 1.20.3
```

Comment:
I *think* you want 1.20 here so it will pull in the minor version and new build will get newer 1.20.x releases. That seems to strict the right balance of "I want the safest new compilers" vs "gimme latest all the time". Note that with only 1.18 before, the new release built with 1.18.10 (the latest (but EOL) 1.18 release). This is the technique we use in IDPE for example. The Edge team chooses microversion pinning for historical reasons, and it creates a lot of churn.

Reply:
I agree with you. Either way, we do not have CD for nifikop builds, though, so this won't help us keep things up to date. However, should we need to make further updates, we won't need the code change.